dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -185,4 +185,7 @@ setHtmlContentTypeOnError.name=Disable text/html content type on error responses
 setHtmlContentTypeOnError.desc=When this property is false, the webcontainer will not set the response's content type header during the error handling process. An application is responsible to set the response's content type. The default value (true) sets the content type to "text/html".
 
 excludeAllHandledTypesClasses.name=Disable returning handlesTypes parameters
-excludeAllHandledTypesClasses.desc=When this property is set to true, during startup, ServletContainerInitializer implementors that use the HandlesTypes annotation do not receive classes that are specified as HandlesTypes parameters.
\ No newline at end of file
+excludeAllHandledTypesClasses.desc=When this property is set to true, during startup, ServletContainerInitializer implementors that use the HandlesTypes annotation do not receive classes that are specified as HandlesTypes parameters.
+
+skipEncodedCharVerification.name=Skip verify the encoded char %23, %2E, %2F, and %5C in request URI
```

Comment:
suggested rewrite:
skipEncodedCharVerification.name=Skip verification of the %23, %2E, %2F, and %5C encoded characters in the request URI